prod/westeurope/internal/api/functions_app_r3/function_app_slot_staging/terragrunt.hcl

```diff
@@ -0,0 +1,158 @@
+dependency "function_app" {
+  config_path = "../function_app"
+}
+
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+dependency "cosmosdb_account" {
+  config_path = "../../cosmosdb/account"
+}
+
+dependency "cosmosdb_database" {
+  config_path = "../../cosmosdb/database"
+}
+
+dependency "storage_account" {
+  config_path = "../../storage/account"
+}
+
+dependency "storage_account_assets" {
+  config_path = "../../../../common/cdn/storage_account_assets"
+}
+
+dependency "storage_account_logs" {
+  config_path = "../../../../operations/storage_account_logs/account"
+}
+
+dependency "storage_container_message-content" {
+  config_path = "../../storage/container_message-content"
+}
+
+dependency "storage_table_subscriptionsfeedbyday" {
+  config_path = "../../storage/table_subscriptionsfeedbyday"
+}
+
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+dependency "subnet_appbackend" {
+  config_path = "../../../appbackend/subnet"
+}
+
+# Common
+dependency "virtual_network" {
+  config_path = "../../../../common/virtual_network"
+}
+
+dependency "application_insights" {
+  config_path = "../../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../../common/key_vault"
+}
+
+dependency "notification_hub" {
+  config_path = "../../../../common/notification_hub"
+}
+
+dependency "notification_queue" {
+  config_path = "../../storage_notifications/queue_push-notifications"
+}
+
+dependency "notification_storage_account" {
+  config_path = "../../storage_notifications/account"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app_slot?ref=v2.0.37"
+}
+
+inputs = {
+  name                       = "staging"
+  resource_group_name        = dependency.resource_group.outputs.resource_name
+  function_app_name          = dependency.function_app.outputs.name
+  function_app_resource_name = dependency.function_app.outputs.resource_name
+  app_service_plan_id        = dependency.function_app.outputs.app_service_plan_id
+  storage_account_name       = dependency.function_app.outputs.storage_account.name
+  storage_account_access_key = dependency.function_app.outputs.storage_account.primary_access_key
+
+  runtime_version = "~3"
```

Comment:
```suggestion
  runtime_version = "~3"
  
  auto_swap_slot_name = "production"
```

Reply:
We need the auto_swap_slot_name = "production" .... 
Something I missed in the old branch

Reply:
thanks, fixed